docs/integrations/google_colab.rst

```diff
@@ -3,38 +3,108 @@
 Neptune-Google Colab Integration
 ================================
 
-You can run experiments on Google Colab and track them with Neptune.
+..
+    Video [1min screencast]
 
-Follow these steps:
+`Google Colab <https://colab.research.google.com/>`_ is a temporary runtime environment. This means you lose all your data (unless saved externally) once you restart your kernel. This is where you can leverage Neptune. By running your experiments on Google Colab and tracking them with Neptune, you can refer to the experiment stats and attributes, even after the Colab kernel has died.
 
-1. Install Neptune client:
+Introduction
+------------
 
-    Go to your first cell in Google Colab and install `neptune-client`:
+This guide will show you how to:
 
-    .. code-block:: Bash
+#. Install ``neptune-client``
+#. Connect Neptune to your notebook and create the first experiment
+#. Log simple metrics to Neptune and explore them in the UI
 
-        pip install neptune-client
+Before you start
+----------------
 
-2. Set Neptune API token:
+Make sure that you have an account with both `Google <https://support.google.com/accounts/answer/27441?hl=en>`_ and `Neptune <https://neptune.ai/register>`_.
 
-    Go to the Neptune web app and `get your API token <../python-api/how-to/organize.html#find-my-neptune-api-token>`_. Set it to the environment variable `NEPTUNE_API_TOKEN`:
+Quickstart
+----------
 
-    .. code-block:: Bash
+|Run on Colab|
 
-        env NEPTUNE_API_TOKEN='your_private_neptune_api_token=='
+Step 1: Install Neptune client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Go to your first cell in Google Colab and install ``neptune-client``:
 
-3. Delete this cell.
+.. code-block:: Bash
 
-    .. warning::
+   pip install neptune-client
 
-        It is very important that you delete this cell, so as not to share your private token with anyone.
+Step 2: Set Neptune API token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-4. Now, run your training script with Neptune:
+#. Go to the Neptune web app and `get your API token <../python-api/how-to/organize.html#find-my-neptune-api-token>`_
 
-    .. code-block:: Python
+   .. image:: https://neptune.ai/wp-content/uploads/get_token.gif
+      :target: https://neptune.ai/wp-content/uploads/get_token.gif
+      :alt: Getting your Neptune API token
+   
+#. Run the code below:
+    
+   .. code-block:: Python
 
-        import neptune
-        neptune.init('USER_NAME/PROJECT_NAME')
+      from getpass import getpass
+      api_token = getpass('Enter your private Neptune API token')
 
-        with neptune.create_experiment():
-            neptune.send_metric('auc', 0.92)
\ No newline at end of file
+#. Enter the token in the input box. This will save your token to ``api_token``
```

Comment:
I wasn't sure what would happen here. 

Could you please add a short 5-10s gif showing that?

Reply:
I created a GIF using a free software (don't think Loom lets you convert screen recordings to GIFs). The quality isn't that great. Let me know what you think

Reply:
Uploaded new video with side-toolbar removed: https://www.loom.com/share/e4a9efe5d723492dac31897aaab9f981?sharedAppSource=team_library